### ISSUE TITLE
Add ability to specify incorrect word overrides

### ISSUE THREAD

**Issue by gmccullo** [streetsidesoftware/vscode-spell-checker#2078]
The idea here is to be able to say certain words should be incorrect in a context. For example, you might want to specify for a particular file or a particular language that some words should be flagged even though would be correct in the larger scope. Example: suppose you wanted to catch any use of contractions in code comments:

```html
<!-- cspell:incorrect didn't wouldn't couldn't shouldn't -->
```

**Comment by Jason3S** [comment]
@gmccullo,

I'm guess you are asking for this enhancement: [Add an in-document cSpell:flagWords directive · Issue #3080 · streetsidesoftware/cspell](https://github.com/streetsidesoftware/cspell/issues/3080)

**Comment by gmccullo** [comment]
> @gmccullo,
> 
> I'm guess you are asking for this enhancement: [Add an in-document cSpell:flagWords directive · Issue #3080 · streetsidesoftware/cspell](https://github.com/streetsidesoftware/cspell/issues/3080)

That does look like the same idea.

**Comment by Jason3S** [comment]
@gmccullo,

This has been done.

**Example:**
```
<!--- cSpell:forbidWords apple -->
<!--- cSpell:flag-words apple -->
```

Note about Markdown comments. If you use `<!---` with three `-`, it will not show up in the resulting HTML.

**Comment by gmccullo** [comment]
Cool. Is "forbidWords" the same as "flag-words"? I just tried it and they both are reported as "Forbidden word."

![image](https://user-images.githubusercontent.com/26315643/201872964-6d9ada63-f0f4-4b26-9bf0-9aa21efe2efc.png)
